### DISCUSSION TITLE
Highlight a point

### DISCUSSION THREAD

**Discussion by Technician13** [cyberbotics/webots#3600] [Q&A]
My webots's version is 2021a, my system is Ubuntu 18.04 , now I have got a destination point for my robot under world coordinate, what should I do to show this point in the simulation interface? For example, use a little green ball or litlle yellow cubic to make this point highlight. Thanks.

**Comment by ad-daniel** [comment]
If you need it to change positions, the easiest approach is probably to add a `Robot` node with a `Shape` child and then pick whatever shape (`Sphere`, `Box`, etc) you like as geometry. Then you can use a supervisor to move the "robot" (the shape) around to whatever position you wish to and change its appearance (color) using also the supervisor functions.
